remote_nodes.sh

```diff
@@ -7,12 +7,13 @@ source ocp_install_env.sh
 
 ACTION=${1:-setup}
 NS=${2:-openshift-machine-api}
-TEARDOWN_PLAYBOOK=/tmp/${REMOTE_CLUSTER_NAME}-teardown-playbook.yml
 IPTABLES=iptables
 
-REMOTE_CLUSTER_NAME=${REMOTE_CLUSTER_NAME:-${CLUSTER_NAME}${NS}}
+APPLY_REMOTE_NODES=${APPLY_REMOTE_NODES:-"true"}
+REMOTE_CLUSTER_NAME=${REMOTE_CLUSTER_NAME:-"${CLUSTER_NAME}spoke"}
 REMOTE_CLUSTER_NUM_MASTERS=${REMOTE_CLUSTER_NUM_MASTERS:-1}
 REMOTE_CLUSTER_NUM_WORKERS=${REMOTE_CLUSTER_NUM_WORKERS:-0}
+REMOTE_IP_STACK=${REMOTE_IP_STACK:-"v4v6"}
```

Comment:
What do you think about using `$IP_STACK` as the default? This should also help us preserve compatibility with the current workflow, if `REMOTE_IP_STACK` is not set.

Reply:
I don't really see an issue with this would make running it locally easier too. :)